Header header

int32 lane_num
float64 ACC_speed

bool ACC_flag
int32 ACC_index_gap
